spokestack/nlu/parsers/integer.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module contains the logic to parse integers from NLU results
+"""
+from typing import Any, Dict, Union
+
+from spokestack.nlu.parsers import maps
+from spokestack.nlu.parsers.digits import DIGIT_SPLIT_RE
```

Comment:
I wouldn't have the parsers import from each other; seems like the word maps and this regex belong in either a utils module or `parsers.__init__`.